packages/css-framework/src/components/_carditem.scss

```diff
@@ -1,6 +1,6 @@
 // .rn-carditem {
```

Comment:
Do we need to keep these commented out styles? Maybe we could remove these?

Reply:
Yeah, I'll delete the card styles. We've only released the frame, so I reckon this is legacy stuff.